.eslintrc.js

```diff
@@ -1 +1,11 @@
-module.exports = require('./index.js');
+const t = [require('./index.js')];
```

Comment:
Why do you import this?

Reply:
`I reverted .eslintrc.js as this code not required (I was forced to disable import/no-unused-modules)
https://github.com/EkoCommunications/EkoESLintConfig/pull/6/files#diff-17ec2f3ce30d31d97dd608a120ada07fR14`

Originally I made this changes to disable import/no-unused-modules for eslint-config-eko.
We have `npm run test` which reads `.eslintrc.js ` and lint package.